README.md

```diff
@@ -7,7 +7,7 @@
 Visit [prometheus.io](https://prometheus.io) for the full documentation,
 examples and guides.
 
-Prometheus is a systems and service monitoring system. It collects metrics
+Prometheus, a [Cloud Native Computing Foundation](https://cncf.io/) project, is a systems and service monitoring system. It collects metrics
```

Comment:
I'm flexible, just figured it would be a nice place to put it.

I have no idea why Travis isn't happy